platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/reconnect/emergency/EmergencyReconnectProtocol.java

```diff
@@ -59,24 +61,21 @@ public class EmergencyReconnectProtocol implements Protocol {
     private final FallenBehindManager fallenBehindManager;
 
     /**
-     * @param threadManager
-     * 		responsible for managing thread lifecycles
-     * @param peerId
-     * 		the ID of the peer we are communicating with
-     * @param emergencyRecoveryManager
-     * 		the state of emergency recovery, if any
-     * @param teacherThrottle
-     * 		restricts reconnects as a teacher
-     * @param stateFinder
-     * 		finds compatible states based on round number and hash
-     * @param reconnectSocketTimeout
-     * 		the socket timeout to use when executing a reconnect
-     * @param reconnectMetrics
-     * 		tracks reconnect metrics
-     * @param reconnectController
-     * 		controls reconnecting as a learner
-     * @param fallenBehindManager
-     *      maintains this node's behind status
+     * Enables submitting platform status actions
+     */
+    private final StatusActionSubmitter statusActionSubmitter;
+
+    /**
+     * @param threadManager            responsible for managing thread lifecycles
+     * @param peerId                   the ID of the peer we are communicating with
```

Comment:
There's a missing `@param` for `notificationEngine`

Reply:
added